docs/mp/health/01_introduction.adoc

```diff
@@ -114,24 +126,360 @@ microservice reports itself as ready.
 |Treats the instance as still starting up; does not check liveness or readiness until the startup probe reports success or times out according to its configuration.
 |===
 
-=== Built-in and Custom Health Checks
 
-==== Built-in Health Checks
-Helidon provides built-in, default checks for each endpoint.
-The built-in liveness checks include various environmental values, such as whether the JVM has detected deadlocks
-or whether there is sufficient heap space. The built-in readiness and startup checks always report `UP`.
+== Configuration
+
+include::../../shared/config/io.helidon.health.HealthSupport.Builder.adoc[]
+
+== Reference
+
+Configuration properties, values and description
+
+|===
+|Property | Values | Description
+
+|health:web-context
+|`String`
+| The endpoint name on which the health context is propagated
+
+|===
+
+
+== Examples
+
+Generate Helidon MP Quickstart project following these <<Instructions, Instruction>>
+
+=== Using the Built-In Health Checks
+
+Helidon has a set of built-in health checks that are enabled to report various
+health check statuses that are commonly used:
+
+* deadlock detection
+* available disk space
+* available heap memory
+
+The following example will demonstrate how to use the built-in health checks.  These examples are all executed
+from the root directory of your project (helidon-quickstart-mp).
+
+[source,bash]
+.Build the application, then run it:
+----
+mvn package
+java -jar target/helidon-quickstart-mp.jar
+----
+
+[source,bash]
+.Verify the health endpoint in a new terminal window:
+----
+> curl http://localhost:8080/health
```

Comment:
You don't have ">" in the snippet above.

Reply:
Removed ">"